website/sidebars.json

```diff
@@ -9,25 +9,25 @@
     "MobX core": [
       "refguide/observable",
       "refguide/action",
-      "refguide/autorun",
-      "refguide/computed"
+      "refguide/computed",
+      "refguide/autorun"
     ],
     "MobX and React": [
-      "react/react-integration"
+      "react/react-integration",
+      "react/react-performance"
```

Comment:
Makes sense to move performance here. But I think it should be marked advanced as well. It kinda is.

Reply:
Understood & marked it as advanced in the menu & 1st heading. While we're at it, menus use the marking without [], whereas all Tip: and headings use [] around it. Do you want all of them to be a certain way or like this?